src/components/product/product-name-creator.test.js

```diff
@@ -0,0 +1,59 @@
+import nameCreatorFactory, { stringifyStack } from './product-name-creator'
+
+describe('stringifyStack', () => {
+  it('verifies create string with product type', () => {
```

Comment:
`verifies` - лишнее слово, давай его уберем и сделаем такое имя:
```suggestion
  it('stringifies stack of 1 item ', () => {
```